test/auth.test.js

```diff
@@ -357,7 +357,7 @@ describe('RO.auth', () => {
           .post(RO.auth.getTokenPath(), { ...postBody })
           .socketDelay(config.timeout + 10)
           .times(3)
-          .reply(200, reply)
+          .replyWithError('ESOCKETTIMEDOUT')
```

Comment:
👍

Reply:
Question: Do the updated tests work with Node v12?

Reply:
tests are passing on `v10.16.3` and `v12.18.4`

Reply:
are there specific versions we should be testing?

Reply:
Good news! I asked because the tests were previously failing on Node 12. Nice work! 🙏